.chalice/policy-prod.json

```diff
@@ -0,0 +1,16 @@
+{
+    "Version": "2012-10-17",
+    "Statement": [
+        {
+            "Action": [
+                "es:ESHttpDelete",
+                "es:ESHttpGet",
+                "es:ESHttpHead",
+                "es:ESHttpPost",
+                "es:ESHttpPut"
+            ],
+            "Effect": "Allow",
+            "Resource": "*"
```

Comment:
Having a policy per stage will be helpful, allowing the ES Resource to be tied to a specific ES Domain going forward, consistent with the config.json -- thanks!

Reply:
OK, these are in the repo. When the different stages resources are better defined they can be edited in those documents.